src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -111,6 +111,13 @@ protected override string GetInputTooltip(int index)
                         AstFactory.BuildExprList(vals)
                     }));
         }
+
+        public event EventHandler MigrationAssistantRequested;
+        public void OnMigrationAssistantRequested(EventArgs e)
```

Comment:
don't think this needs to be `public`, if it works with `private` , let's make it that

Reply:
i need this as public as i have to invoke it from the `ScriptEditorWindow` which is on `PythonNodeModelsWpf`

Reply:
then we rename this to be `RequestCodeMigration()` as it's used to invoke the event, rather than what happens when it's invoked